src/DssSpellCollateralOnboarding.sol

```diff
@@ -89,38 +74,26 @@ contract DssSpellCollateralOnboardingAction {
             })
         );
         DssExecLib.setStairstepExponentialDecrease(
-            MCD_CLIP_CALC_ETHSTETH_A,
+            MCD_CLIP_CALC_XXX_A,
             90 seconds,
             9900
         );
         DssExecLib.setIlkAutoLineParameters(
-            "CRVV1ETHSTETH-A",
+            "XXX-A",
             3 * MILLION,
             3 * MILLION,
             8 hours
         );
 
-        // Extra for crop-join - need to authorize the join adapter with the cropper
-        DssExecLib.authorize(MCD_JOIN_ETHSTETH_A, CROPPER);
-
-        // Need to whitelist the stETH Medianizer with the wstETH Medianizer
-        DssExecLib.addReaderToWhitelistCall(WSTETH_MEDIAN, STETH_PRICE);
-
         // Whitelist OSM - normally handled in addNewCollateral, but Curve LP Oracle format is not supported yet
-        DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(0), PIP_ETHSTETH);
-        DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(1), PIP_ETHSTETH);
+        // DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(0), PIP_ETHSTETH);
+        // DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(1), PIP_ETHSTETH);
 
         // ChainLog Updates
-        DssExecLib.setChangelogAddress("CDP_REGISTRY", CDP_REGISTRY);
-        DssExecLib.setChangelogAddress("MCD_CROPPER", CROPPER);
-        DssExecLib.setChangelogAddress("MCD_CROPPER_IMP", CROPPER_IMP);
-        DssExecLib.setChangelogAddress("PROXY_ACTIONS_CROPPER", PROXY_ACTIONS_CROPPER);
-        DssExecLib.setChangelogAddress("PROXY_ACTIONS_END_CROPPER", PROXY_ACTIONS_END_CROPPER);
-        
-        DssExecLib.setChangelogAddress("CRVV1ETHSTETH", ETHSTETH);
-        DssExecLib.setChangelogAddress("PIP_CRVV1ETHSTETH", PIP_ETHSTETH);
-        DssExecLib.setChangelogAddress("MCD_JOIN_CRVV1ETHSTETH_A", MCD_JOIN_ETHSTETH_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CRVV1ETHSTETH_A", MCD_CLIP_ETHSTETH_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_CRVV1ETHSTETH_A", MCD_CLIP_CALC_ETHSTETH_A);
+        DssExecLib.setChangelogAddress("XXX", XXX);
+        DssExecLib.setChangelogAddress("PIP_XXX", PIP_XXX);
+        DssExecLib.setChangelogAddress("MCD_JOIN_XXX_A", MCD_JOIN_XXX_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_XXX_A", MCD_CLIP_XXX_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_XXX_A", MCD_CLIP_CALC_XXX_A);
```

Comment:
I would comment all these examples out just to be safe

Reply:
done